Fortran/exercise2/league_sim.f90

```diff
@@ -92,4 +39,5 @@ subroutine print_results(team_names, points)
     end do
 end subroutine print_results
```

Comment:
The `print_results` subroutine is present in both `league_sim.f90` and `league_sim_mod.f90`, do we want to import it from the module in the main script?